### PR TITLE
Add basemap to timelapse-video and fix firefox issue

### PR DESCRIPTION
**Pull request by sdobbert**
The timelapse-video process now creates the video from a basemap and the transparent temperature layer.
Additionally, the video is now supported in firefox (https://github.com/klips-project/klips-sdi/issues/161).
image

### CODE REVIEW

File: easy-to-use-api/klips-timelapse-video-api/package.json [hunk 1]
```diff
@@ -22,13 +22,13 @@
     "@testing-library/dom": "^9.3.3",
     "@testing-library/jest-dom": "^6.1.4",
     "@types/jest": "^29.5.10",
-    "@typescript-eslint/eslint-plugin": "^6.13.1",
+    "@typescript-eslint/eslint-plugin":"5.62.0",
```
Comment (ahennr) [inline easy-to-use-api/klips-timelapse-video-api/package.json:25]: Why did you need to downgrade `eslint-plugin` and `typescript`?

**Review by ahennr — APPROVED** [review]

**Review by weskamm — APPROVED** [review]

File: easy-to-use-api/klips-timelapse-video-api/package.json (hunk 1 above)
Comment (weskamm) [reply easy-to-use-api/klips-timelapse-video-api/package.json:25]: good question

Comment (sdobbert) [reply easy-to-use-api/klips-timelapse-video-api/package.json:25]: There was a dependency conflict with the newer version: 

> Could not resolve dependency:
> 3.303 npm ERR! dev @typescript-eslint/eslint-plugin@"^6.13.1" from the root project
